docs/Air.md

```diff
@@ -239,3 +239,32 @@ Request for the ticket information for all tickets in PNR.
 | pnr | `String` | 1G PNR. |
 
 **See: <a href="../examples/Air/getTickets.js">getTickets example</a>**
+
+
+<a name="searchBookingsByPassengerName"></a>
+### .searchBookingsByPassengerName(params)
+> Requires Terminal access enabled in uAPI. See [TerminalService](Terminal.md)
+Request for the ticket information.
+
+**Returns**: `Promise` with next shape `{ type: 'list' | 'pnr', data: Array | String }`

```

Comment:
Why do we need such a complication?
Why don't we always return an array? (Of one element if the only booking is found)